toolbox/README.md

```diff
@@ -0,0 +1,28 @@
+#  Rook Toolbox
+The rook toolbox is a container with common tools used for rook debugging and testing.  All packages in the toolbox can be seen in the [Dockerfile](Dockerfile)
+
+## Container Linux by CoreOS
+To use the rook toolbox on CoreOS, first add the following values to the toolbox config file:
+```
+cat >~/.toolboxrc <<EOL
+TOOLBOX_DOCKER_IMAGE=quay.io/rook/toolbox
+TOOLBOX_DOCKER_TAG=latest
+EOL
+```
+
+Then launch the toolbox as usual:
+```
+toolbox
+```
+
+### Ceph Tools
+To use the ceph tools from a rook host, launch the toolbox and run the following command:
+```
+toolbox --bind=/var/lib/rook:/var/lib/rook
+ROOK_CONFIG=`find /var/lib/rook -regex '.*mon[0-9]+/.*\.config' | head -1`
```

Comment:
an easy to implement but better experience here would be a simple entrypoint script that gets or generates a rook/ceph cluster config file into the default spot that `ceph` and `rados` commands look for (i believe `/etc/ceph`) in the container, so the user doesn't have to deal with anything beyond running the container and then running `ceph` or `rados` with no additional args